HTML/Ghibli.html

```diff
@@ -0,0 +1,74 @@
+  </li>
+  <li class="li" style="display: block; float: left; margin-right:27px; font-size:20px;">
+    <a href="Git.html" style="color:white; text-decoration: none;">Git</a>
+  </li>
+  <li style="display: block; float: left; margin-right:27px; ">
+    <a href="HTML.html" style="color:white; text-decoration: none; font-size:20px; ">HTML</a>
+  </li>
+  <li style="display: block; float: left; margin-right:27px; font-size:20px;">
+    <a href="Javascript.html" style="color:white; text-decoration: none;">Javascript</a>
+  </li>
+  <li style="display: block; float: left; margin-right:27px; font-size:20px;">
+    <a href="Ghibli.html" style="color:white; text-decoration: none;">Studio Ghibli</a>
+  </li>
+</ul>
+
+<h2 style="margin-left: 50px"> Studio Ghibli</h2>
+<div style="margin-left: 400px;">
+<img src="https://i.pinimg.com/originals/60/60/8a/60608ad6bde1038bf3e2185f312c090d.png" width="1000px" >
```

Comment:
:hammer_and_wrench: Bitte immer ein 'alt' attribute beim `img` Tags inkludieren

![image](https://user-images.githubusercontent.com/68048149/184291214-bc5f921b-f53c-40d2-b90a-160515e53698.png)

```suggestion
<img alt="totoro" src="https://i.pinimg.com/originals/60/60/8a/60608ad6bde1038bf3e2185f312c090d.png" width="1000px" >
```